src/static/index.html

```diff
@@ -0,0 +1,324 @@
+            align-items: center;
+            height: auto;
+            font-weight: bold;
+            font-size: 36px;
+            height: 10vh;
+        }
+
+        .instructions {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            font-size: 16px;
+            width: 100%;
+            height: 5vh;
+        }
+
+        .command {
+            --tw-bg-opacity: 1;
+            --tw-text-opacity: 1;
+            width: 20rem;
+            font-size: 0.75rem;
+            line-height: 1rem;
+            color: rgba(39, 174, 96, var(--tw-text-opacity));
+            border-width: 1px;
+            border-radius: 0.5rem;
+            background-color: rgba(51, 51, 51, var(--tw-bg-opacity));
+            margin-top: 0.75rem;
+            padding-left: 1rem;
+            height: 5vh;
+        }
+
+        .table-container {
+            height: 70vh;
+            max-width: 90vw;
+            overflow: scroll;
+            margin-top: 24px;
+            border: 1px solid #D1CFD7;
+        }
+
+        .table {
+            --tw-border-opacity: 1;
+            position: relative;
+            border-collapse: collapse;
+
+        }
+
+        .table-head {
+            background-color: #F7F7FA;
+        }
+
+        .circle-green {
+            width: 10px;
+            height: 10px;
+            border-radius: 50%;
+            background-color: #4CD964;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+
+        .circle-red {
+            width: 10px;
+            height: 10px;
+            border-radius: 50%;
+            background-color: #FF4E3D;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+
+        th {
+            padding: 24px;
+            text-align: center;
+        }
+
+        td {
+            border: 1px solid #D1CFD7;
+        }
+
+        tr:nth-child(even) {
+            background-color: #F7F7FA;
+        }
+
+        td:hover {
+            background-color: #E0E0E0;
+        }
+
+        th {
+            /* Top Row */
+            background-color: #D1CFD7;
+            position: sticky;
+            top: 0px;
+            z-index: 9;
+        }
+
+        th:first-of-type {
+            /* Top Left Cell */
+            left: 0;
+            z-index: 10;
+        }
+
+        tbody tr td:first-of-type {
+            /* Left Column */
+            background-color: #D1CFD7;
+            position: sticky;
+            max-width: 20vw;
+            overflow-wrap: break-word;
+            left: 0px;
+            text-align: left;
+            padding: 16px;
+            z-index: 5;
+        }
+    </style>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script>
+        const executeCommand = async (command) => {
+            const res = await axios.post('http://localhost:3000/command', {
```

Comment:
the 3000 here is hardcoded, so it won't match if i override it to something else.

wouldn't this work?
```suggestion
            const res = await axios.post('/command', {
```

Reply:
Oops, I missed this, fixing right now